smarts/core/trap_manager.py

```diff
@@ -145,6 +145,7 @@ def largest_vehicle_plane_dimension(vehicle):
                 ),
             )
             for v_id in sorted_vehicle_ids:
+                break
```

Comment:
this block of code is responsible for capturing any colliding vehicles with the agent. This causing another bug with SUMO. Since vehicles in history records shouldn't collide there shouldn't be a need to capture. Added a break here to avoid capturing